File: examples/location/read/node/getLocationById.js
const { LocationService } = require("micro-js-client/location");

// Read an entity by its ID
async function getLocationById() {
  let locationService = new LocationService(process.env.MICRO_API_TOKEN);
  let rsp = await locationService.read({
    id: "1",
  });
  console.log(rsp);
}

getLocationById();

File: examples/go/location/read/getLocationById.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/location"
)

// Read an entity by its ID
func GetLocationById() {
	locationService := location.NewLocationService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := locationService.Read(&location.ReadRequest{
		Id: "1",
	})
	fmt.Println(rsp, err)
}
